Cloth, Rugs, Advancements and stuff

Shnupbups opened the pull request:
- renamed Old Bandage to Old Cloth
- Old Cloth can be used to craft White Cloth blocks, which can be dyed into any other colour
- Cloth blocks can be used in place of Wool to craft things like Beds and Banners
- Rugs also exist, which are just Carpets made of Cloth

horrible Cloth block textures made by yours truly

EDIT: further changes are listed below

Shnupbups committed:
Changed Old Bandage to Cloth

Shnupbups committed:
Merge remote-tracking branch 'upstream/master'

Shnupbups committed:
Update fr_fr.json

Shnupbups committed:
Cloth blocks and rugs

You can now craft the old cloth from mummies into cloth blocks, which can be used as a substitute for wool and dyed in all 16 colours. They also have 'rugs' which are just their version of carpets. Complete with a garbage texture from your truly.

LemmaEOF reviewed (approved):
hmm, would it be better to add recipes that use tags for all the cloth stuff, so you can make things using a combination of wool and cloth? either way, this is looking good.

Shnupbups committed:
Attempted to fix issues with models

fixed #missing texture stuff and fixed uv mappings on pumpkin pie blocks

Shnupbups committed:
more uv fixes

Shnupbups committed:
Update pumpkin_pie_one_quarter.json

Shnupbups committed:
Some stuff

- Added Advancements to guide the player into The Hallow
- Fixed teleportation to The Hallow not always putting the Player in a safe position
- Furnaces can now be crafted with Tainted Cobblestone
- The 2 Hallow Charm recipes are now grouped together
- Fixed the Hallowed Gate validity check not checking the full ring
- Fixed the 'gate incomplete' message having the wrong translation key
- Changed the 'gate incomplete' message to show in the Action Bar
- Added a message for attempting to use the Hallowed Gate outside the Overworld
- Added 'grey' spelling for Cloth and Rugs to Englishes other than US

Shnupbups commented:
Changes in latest commit:
- Added Advancements to guide the player into The Hallow
- Fixed teleportation to The Hallow not always putting the Player in a safe position
- Furnaces can now be crafted with Tainted Cobblestone
- The 2 Hallow Charm recipes are now grouped together
- Fixed the Hallowed Gate validity check not checking the full ring
- Fixed the 'gate incomplete' message having the wrong translation key
- Changed the 'gate incomplete' message to show in the Action Bar
- Added a message for attempting to use the Hallowed Gate outside the Overworld
- Added 'grey' spelling for Cloth and Rugs to Englishes other than US

Shnupbups committed:
Fixed various issues with the Tombstone

Added localisation, edited the models and texture for the tombstone

Shnupbups committed:
added some noise to cloth textures

vini2003 reviewed (approved):
All is great!
